examples/random_forest.py

```diff
@@ -25,7 +25,6 @@
     clf,
     param_distributions,
     n_iter=3,
-    max_iters=10,
```

Comment:
Why did we remove `max_iters` here? Not that it makes a huge difference in terms of showing the example but jw.

Reply:
We didn't activate `early_stopping` in this example, so setting `max_iters` doesn't do anything. I removed it to prevent confusion of how this parameter is used